src/targets/gpu/perfdb.cpp

```diff
@@ -27,6 +27,9 @@
 #include <migraphx/stringutils.hpp>
 #include <migraphx/permutation.hpp>
 #include <fstream>
+#include <mutex>
+
+static std::mutex g_db_mutex; // NOLINT
```

Comment:
This variable should go in the `query_miopen_db` function.

Reply:
Makes sense, it's already `static`.